tests/unit/sdk/test_node.py

```diff
@@ -1789,7 +1794,7 @@ async def test_update_input_data_empty_relationship(
         "data": {
             "id": "llllllll-llll-llll-llll-llllllllllll",
             "name": {"value": "DFW"},
-            # "primary_tag": None,
+            "primary_tag": None,
```

Comment:
I think having these should be alright. This does not seem to have a negative impact.